### DISCUSSION TITLE
1.2.5, win10 with solid color desktop background gets background activated with new 1.2.5 background removal test button

### DISCUSSION THREAD

**Discussion by aabittner** [rustdesk/rustdesk#8254] [General]
### Bug Description

just tried the new preview (near release) of 1.2.5 rustdesk on windows, win-exe installer, on a win10 professional x64 system.

in the rustdesk settings I noticed the remove-background feature that i didnt recognize so far from the past. anyhow my windows10 system being testing on, does have only solid color background for its desktop.

the rustdesk 1.2.5 feature to remove background in a rustdesk session, seems to re-toggle the once-set (by the OEM manufacturer or some microsoft delivered default win10 style background image) background image.

maybe the code of rustdesk needs to check if a background image is being used in the first place before changing settings of the user.

thank you. 

### How to Reproduce

after using this test button in the settings, the not-set background image stays after this testing button. so its reversed to the expected (wanted) behavior for users who dont have actually set a background pixel image in the first place.

### Expected Behavior

see above.

### Operating system(s) on local side and remote side

windows 10, professional, x64, non-english

### RustDesk Version(s) on local side and remote side

1.2.5 preview of today (2024-06-03)

### Screenshots

-

### Additional Context

thank you.

**Comment by rustdesk** [comment]
@21pages

**Comment by 21pages** [comment]
Your original wallpaper is solid color, and it doesn't reset back to the original wallpaper after test? Or your original wallpaper is solid color, RustDesk shouldn't use another solid color during session?

https://github.com/rustdesk/rustdesk/assets/14891774/f0017cce-151f-4b02-a19b-a258c4b18a4f

https://github.com/rustdesk/rustdesk/assets/14891774/f05e876b-ffb6-48e9-9440-8a348ed1db2f

**Comment by rustdesk** [comment]
@aabittner please upload a video, since your english is really hard to understand. :(

**Comment by aabittner** [comment]
the original users background in win10 is a solid color. the rustdesk setting to remove the background looks new to me so I tested that button in its settings (to remove background image, even though I didnt have one set)

when using that test, it inverts the situation, it SETS an ancient old background although I didnt have that for years and this background image STAYS when i exit the settings area of rustdesk.

this makes no sense. its a software bug in logics of rustdesk i would argue.
